src/custom-element.js

```diff
@@ -1558,6 +1567,13 @@ function createBaseCustomElementClass(win) {
      */
     toggleFallback(state) {
       assertNotTemplate(this);
+      const resourceState = this.getResourceState_();
+      // Do not show fallback if element has not been scheduled layout yet
+      if (state && (resourceState == ResourceState.NOT_BUILT ||
```

Comment:
can you rename `state` to `enabled` so that it can be differentiated from `resourceState`